draft-ietf-taps-impl.md

```diff
@@ -678,22 +681,22 @@ Connection Object:
 : TCP connections between two hosts map directly to Connection objects.
 
 Initiate:
-: Calling `Initiate` on a TCP Connection causes it to reserve a local port, and send a SYN to the Remote Endpoint.
+: CONNECT.TCP. Calling `Initiate` on a TCP Connection causes it to reserve a local port, and send a SYN to the Remote Endpoint.
 
 InitiateWithSend:
-: Early idempotent data is sent on a TCP Connection in the SYN, as TCP Fast Open data.
+: CONNECT.TCP with parameter "user message". Early idempotent data is sent on a TCP Connection in the SYN, as TCP Fast Open data.
 
 Ready:
-: A TCP Connection is ready once the three-way handshake is complete.
+: `Initiate` or `InitiateWithSend` returns without an error, i.e. the three-way handshake is complete.
 
 InitiateError:
-: TCP can throw various errors during connection setup. Specifically, it is important to handle a RST being sent by the peer during the handshake.
+: Failure of CONNECT.TCP. TCP can throw various errors during connection setup. Specifically, it is important to handle a RST being sent by the peer during the handshake.
 
 ConnectionError:
-: Once established, TCP throws errors whenever the connection is disconnected, such as due to receive a RST from the peer; or hitting a TCP retransmission timeout.
+: Once established, TCP throws errors whenever the connection is disconnected, such as due to receive a RST from the peer; or hitting a TCP retransmission timeout. This can lead to ABORT-EVENT.TCP or TIMEOUT.TCP events.
```

Comment:
This doesn't lead to the ABORT-EVENT.TCP or TIMEOUT.TCP events I think-- rather the ABORT-EVENT.TCP or TIMEOUT.TCP events lead to the ConnectionError event!

Reply:
Of course, you're right! Thanks, I'll fix this!